src/ngraph/runtime/cpu/pregenerated_src/cpu_cg_runtime_context.hpp

```diff
@@ -70,4 +96,26 @@ extern "C" void destroy_cg_ctx(CPURuntimeContextCG* cg_ctx)
 {
     delete cg_ctx;
 }
+
+extern "C" void set_memory_ptr_cg(CPURuntimeContextCG* cg_ctx,
```

Comment:
dont think we need extern "C" on these functions since they are only going to get called from within the generated code.